include/wlr/types/wlr_xdg_shell.h

```diff
@@ -392,6 +392,14 @@ void wlr_xdg_surface_get_geometry(struct wlr_xdg_surface *surface,
 void wlr_xdg_surface_for_each_surface(struct wlr_xdg_surface *surface,
 		wlr_surface_iterator_func_t iterator, void *user_data);
 
+/**
+ * Call `iterator` on each popup surface and subsurface in the xdg-surface
```

Comment:
Can we make it clearer that this won't iterate over subsurfaces of the root xdg-surface? Maybe s/subsurface/popup's subsurface/?

Reply:
Done, it sounds a little funny now but it's certainly more clear that way.